.github/workflows/test-runner-nightly-integration.yml

```diff
@@ -0,0 +1,91 @@
+
+    services:
+      postgres:
+        image: postgres:12.3
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Initialize Postgres DB
+        env:
+          PGPASSWORD: postgres
+        run: psql -h 127.0.0.1 -U postgres -f ./local-dev/local-postgres-init.sql
+      - name: Set up AdoptOpenJDK 11
+        uses: joschi/setup-jdk@v2
+        with:
+          java-version: 11
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle') }}
+          restore-keys: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      # These steps aren't needed for unit tests
+      - name: Get Vault token
+        if: matrix.gradleTask != 'unitTest'
+        id: vault-token-step
+        env:
+          VAULT_ADDR: https://clotho.broadinstitute.org:8200
+        run: |
+          VAULT_TOKEN=$(docker run --rm --cap-add IPC_LOCK \
+            -e "VAULT_ADDR=${VAULT_ADDR}" \
+            vault:1.1.0 \
+            vault write -field token \
+              auth/approle/login role_id=${{ secrets.VAULT_APPROLE_ROLE_ID }} \
+              secret_id=${{ secrets.VAULT_APPROLE_SECRET_ID }})
+          echo ::set-output name=vault-token::$VAULT_TOKEN
+          echo ::add-mask::$VAULT_TOKEN
+      - name: Set permissions and render test configurations
+        run: |
+          chmod +x render_config.sh
+          ./render_config.sh ${{ steps.vault-token-step.outputs.vault-token }} dev
+          chmod +r rendered
+
+      # Only integration tests need a running server
+      - name: Run local server (in background)
+        run: ./gradlew bootRun &
+      - name: Wait for local server to start
+        # With a timeout of 60 seconds, try to connect to localhost:8080 every 1 second.
+        run: timeout 60 bash -c 'until echo > /dev/tcp/localhost/8080; do sleep 1; done'
+
+      - name: Build the WSM client and run the Basic Integration test suite
+        run: |
+          echo "Building WSM client library"
+          ./gradlew :workspace-manager-client:clean :workspace-manager-client:assemble
+          cd workspace-manager-clienttests
+          export ORG_GRADLE_PROJECT_workspacemanagerclientjar=$(find .. -type f -name "workspace-manager-client*.jar")
+          echo "ORG_GRADLE_PROJECT_workspacemanagerclientjar = ${ORG_GRADLE_PROJECT_workspacemanagerclientjar}"
+          echo "Test Runner config"
+          ./render-config.sh ${{ steps.vault-token-step.outputs.vault-token }}
```

Comment:
The `workspace-manager-clienttests` project has its own copy of render-config since it started as a standalone project, it also rendered the test runner SA key that the parent service render config doesn't care about. I put this in the Google [doc](https://docs.google.com/document/d/1gYqlJTpC4C4S1K0pYikeNJx-HWNZAZJYybinmuqcSTI/edit#) for future discussions.